aspnetcore/migration/mvc.md

```diff
@@ -41,7 +41,7 @@ Create a new *empty* ASP.NET Core web app with the same name as the previous pro
 
 ## Configure the site to use MVC
 
-* Install the `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore.StaticFiles` NuGet packages.
+* By Default, the ASP.NET Core MetaPackage is added to the project, called `Microsoft.AspNetCore.All`. This package contains packages like `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore.StaticFiles`.
```

Comment:
By Default -> By default
MetaPackage -> metapackage

In 2.0 we use `Microsoft.AspNetCore.All` but in 2.1 we use `Microsoft.AspNetCore.App`. 

However, the metapackages are only used when running on .NET Core. When running on the .NET Framework you still reference individual packages.

Reply:
I assumed this doc would need to get updated when 2.1 is released. Should we add a note about 2.1 preview?

Reply:
Just make sure you're tracking this for 2.1.

Reply:
@scottaddie FYI, this doc will need to be upgraded with 2.1